src/attributes/type_system.md

```diff
@@ -127,6 +127,14 @@ match message {
 }
 ```
 
+It's also impossible to cast non-exhaustive types from foreign crates.
```

Comment:
Just a minor wording suggestion. I'm not sure why, but "impossible" doesn't seem to quite fit here for me. It is possible, it just results in an error. I kinda view things that are "impossible" as fundamentally not even possible to try.

```suggestion
It's also not allowed to cast non-exhaustive types from foreign crates.
```

Reply:
Instant answer!

I agree with your logic here. The MR in the rust repository forbade this cast, so it was possible beforehand.